### ISSUE TITLE
ic-include bug for array of elements

### ISSUE THREAD

**Issue by styxyk** [LeadDyno/intercooler-js#107]
When using ic-include attribute to submit a form which contains a group of checkboxes (an array of checkbox inputs) the submitted values will be duplicated.

```
<form id="form">
<div id="div">
  <input type=checkbox name=checkarray[] value=aaa checked> 
  <input type=checkbox name=checkarray[] value=bbb checked> 
  <input type=checkbox name=checkarray[] value=ccc> 
  <button ic-post-to="results.php" ic-target="#res" ic-include="#form"> send </button>
</div>
</form>
<div id="res"></div>
```

The result will be:  [checkarray] => Array ( [0] => aaa [1] => bbb [2] => aaa [3] => bbb )

However if we change ic-include="#form" to ic-include="#div" the result will be correct:  [checkarray] => Array ( [0] => aaa [1] => bbb )

**Comment by styxyk** [comment]
sorry, I didn't have to add ic-include attribute at all.

**Comment by carsongross** [comment]
:)

Cheers,
Carson

On July 25, 2016 at 9:07:13 AM, styxyk (notifications@github.com) wrote:

Closed #107.
